eng/common/scripts/Save-Package-Properties.ps1

```diff
@@ -58,11 +65,6 @@ function SetOutput($outputPath, $incomingPackageSpec) {
     $outputObject.DevVersion = $incomingPackageSpec.Version
   }
 
-  # Set file paths to relative paths
-  $outputObject.DirectoryPath = GetRelativePath $outputObject.DirectoryPath
-  $outputObject.ReadMePath = GetRelativePath $outputObject.ReadMePath
-  $outputObject.ChangeLogPath = GetRelativePath $outputObject.ChangeLogPath
-
   Set-Content `
     -Path $outputPath `
     -Value (ConvertTo-Json -InputObject $outputObject -Depth 100)
```

Comment:
GetRelativePath ends up being dependent on the current working directory when $path is already relative, which is definitely not something we want to be dependent on. We either need to switch to a different mechanism or do a check to see if the path is already relative first or a simple starts with check before calling it.

Reply:
Checking whether the path is relative is by far the superior solution. Thanks!